lib/azuremetadata/azuremetadata.py

```diff
@@ -144,6 +144,14 @@ def _make_request(url):
                     data = data.decode('utf-8')
                 return json.loads(data)
             except urllib.error.HTTPError as e:
+                # remove this case when versions API
+                # endpoint includes license type
```

Comment:
I do not understand the comment and what the versiosn API has to do with licenses

Reply:
it is a reminder, remove that code that handle `no_api` when the versions endpoint returns the right data